index.rst

```diff
@@ -265,7 +265,7 @@ Key Resources
     * `Meta tracker <http://psf.upfronthosting.co.za/roundup/meta>`_ (issue
       tracker for the issue tracker)
     * :doc:`experts`
-    * `Firefox search engine plug-in`_
+    * Firefox search engine plug-in
```

Comment:
If the plug-in is gone, then there's no reason to keep this line around.

Reply:
Oh, @brettcannon suggested on the issue that the reference should be removed. 

So, what should I do?

Reply:
Both this line and the link should be removed.

Reply:
@ezio-melotti fixed :)